p4-16/psa/PSA.mdk

```diff
@@ -950,41 +940,33 @@ used in the second pass. See Section [#sec-open-metadata] for the
 discussion.
 
 A PSA implementation provides a configuration bit `resubmit` to the
-PRE to enable the resubmission mechanism. If `resubmit` is set to
-true, the resubmission mechanism resends the original packet with the
-optional attached metadata. If `resubmit` is set to false, the
-resubmission mechanism is disabled and the original packet is not
-resubmitted even if the `add_metadata` method is invoked in the
-deparser.
-
-```
-[INCLUDE=psa.p4:Resubmit_extern]
-
-```
+PRE to enable the resubmission mechanism. If true, the original packet
+is resubmitted with the optional resubmit metadata. If false, the
+resubmission mechanism is disabled and the value assigned to the
+`resubmit_meta` is ignored.
```

Comment:
I think the way the proposal for carrying metadata is now, it would be more precise to say "and no assignments to `resubmit_meta` should be performed"